internal/cli/atlas/data_lake_create.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package atlas
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type DataLakeCreateOpts struct {
+	cli.GlobalOpts
+	store store.DataLakeCreator
+	name  string
+}
+
+func (opts *DataLakeCreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DataLakeCreateOpts) Run() error {
+	createRequest := mongodbatlas.DataLakeCreateRequest{
+		Name: opts.name,
+	}
```

Comment:
[non-blocking] you could do this in the store and pass just the name to the store given that you only create by name but don't feel like you need to change this

Reply:
👍  I can follow whatever we do normally, is there a preference?

Reply:
for single params we prefer to do it on the store, reduces how much we expose of the api to the cli

another thing we've done for more complex creates (loads of flags) is to have a `opts.newAPIType()` where we encapsulate this, but for this one so simple store or how you have works fine

Reply:
(I can go back to this when I look at update)